sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.Metrics;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    internal static class Statsbeat
+    {
+        private static readonly Meter s_myMeter = new("AttachStatsBeatMeter", "1.0");
+
+        private const string StatsBeat_ConnectionString = "<StatsBeat_ConnectionString>";
+
+        internal const int AttachStatsBeatInterval = 86400000;
+
+        static Statsbeat()
+        {
+            s_myMeter.CreateObservableGauge("AttachStatsBeat", () => GetAttachStatsBeat());
+
+            // Configure for attach statsbeat which has collection
+            // schedule of 24 hrs == 86400000 milliseconds.
```

Comment:
Do we have a rough idea regarding the percentage of processes that are going to live above 24 hours?

Reply:
not at the moment. I will follow up to see if we can get this information from our rp integrations.

Reply:
maybe call forceflush at startup, so we ensure stats is sent once. and then if the process lives long enough, then every 24 hours ..

Reply:
I have added a todo for now to follow up with the spec on behavior in case of app exiting before 24hr duration